dev-infrastructure/templates/global-grafana.bicep

```diff
@@ -0,0 +1,58 @@
+@description('The global msi name')
+param globalMSIName string
+
+@description('Metrics global Grafana name')
+param grafanaName string
+
+@description('The admin group principal ID to manage Grafana')
+param grafanaAdminGroupPrincipalId string
+
+resource ev2MSI 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
+  name: globalMSIName
+}
+
+// Azure Managed Grafana Workspace Contributor: Can manage Azure Managed Grafana resources, without providing access to the workspaces themselves.
+// https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/monitor#azure-managed-grafana-workspace-contributor
+var grafanaContributor = '5c2d7e57-b7c2-4d8a-be4f-82afa42c6e95'
+
+// Grafana Admin: Perform all Grafana operations, including the ability to manage data sources, create dashboards, and manage role assignments within Grafana.
+// https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/monitor#grafana-admin
+var grafanaAdminRole = '22926164-76b3-42b3-bc55-97df8dab3e41'
+
+var grafanaAdminGroup = {
+  principalId: grafanaAdminGroupPrincipalId
+  principalType: 'group'
+}
+
+resource grafana 'Microsoft.Dashboard/grafana@2023-09-01' = {
+  name: grafanaName
+  location: resourceGroup().location
+  sku: {
+    name: 'Standard'
+  }
+  identity: {
+    type: 'SystemAssigned'
+  }
```

Comment:
Could I request that you add `zoneRedundancy` here and get the desired value based on zone redundancy availability like we've done for other resources? See https://issues.redhat.com/browse/ARO-14959

Reply:
done